state/machine.go

```diff
@@ -2212,6 +2212,59 @@ func (m *Machine) CompleteUpgradeSeries() error {
 	return nil
 }
 
+// CompleteUnitUpgradeSeries notifies units and machines that an upgrade series is
+// ready for its "completion" phase.
+func (m *Machine) CompleteUnitUpgradeSeries() error {
```

Comment:
The name on this is confusing, as it sound like you are completing the upgrade series step, not starting the "completion" step.
StartUnitUpgradeSeriesCompletionPhase() ?